src/use-picture-in-picture.ts

```diff
@@ -57,47 +58,48 @@ export default function usePictureInPicture(
         !isPictureInPictureDisabled(videoRef.current)
     )
 
-    if (
-      onEnterPictureInPicture &&
-      typeof onEnterPictureInPicture === 'function'
-    ) {
-      videoRef.current.addEventListener(
-        'enterpictureinpicture',
-        onEnterPictureInPicture
-      )
-    }
-    if (
-      onLeavePictureInPicture &&
-      typeof onLeavePictureInPicture === 'function'
-    ) {
-      videoRef.current.addEventListener(
-        'leavepictureinpicture',
-        onLeavePictureInPicture
-      )
-    }
-
-    return () => {
-      if (videoRef.current === null) {
-        return
-      }
+    function _onEnterPictureInPicture(e: EnterPictureInPictureEvent) {
       if (
         onEnterPictureInPicture &&
         typeof onEnterPictureInPicture === 'function'
       ) {
-        videoRef.current.removeEventListener(
-          'enterpictureinpicture',
-          onEnterPictureInPicture
-        )
+        // @ts-ignore
+        onEnterPictureInPicture(e)
       }
+      togglePictureInPicture(true)
+    }
+    function _onLeavePictureInPicture(e: Event) {
```

Comment:
Since the `leavepictureinpicture` event doesn't use the `LeavePictureInPictureEvent` type, at least in my environment, I'm just using `Event` here. lmk if I should remove `LeavePictureInPictureEvent` above